### PR TITLE
Mappb list added

### PR DESCRIPTION
**Pull request by wfkincaid**

### CODE REVIEW

**Comment by wfkincaid** [comment]
need to add descriptions to docstring

File: pyspecdata/plot_funcs/pcolormesh.py [hunk 1]
```diff
@@ -36,10 +37,12 @@ def pcolormesh(
         because matplotlib doesn't let you call sharex/sharey more than once,
         you need then to tell it not to handle the axis sharing, and to it yourself
         outside this routine.
+    mappable_list: list of
```
Comment (jmfranck) [inline pyspecdata/plot_funcs/pcolormesh.py:40]: please fill out -- see example [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html)

**Review by jmfranck — CHANGES_REQUESTED** [review]

File: pyspecdata/plot_funcs/pcolormesh.py [hunk 2]
```diff
@@ -19,12 +19,12 @@ def pcolormesh(
...
     ==========
-    fig: matplotlib figure object
+    fig : matplotlib figure object
     shading: str (default 'nearest')
         the type of shading to pass to matplotlib pcolormesh
-    ax1: matplotlib axes object
+    ax1 : matplotlib axes object
         where do you want the left plot to go?
-    ax2: matplotlib axes object
+    ax2 : matplotlib axes object
```
Comment (jmfranck) [inline pyspecdata/plot_funcs/pcolormesh.py:27]: please note that my format for numpy docstrings wasn't quite to spec, so I'm fixing it moving forward (that's what I'm doing here)

File: pyspecdata/plot_funcs/pcolormesh.py [hunk 3]
```diff
@@ -37,13 +37,16 @@ def pcolormesh(
         because matplotlib doesn't let you call sharex/sharey more than once,
         you need then to tell it not to handle the axis sharing, and to it yourself
         outside this routine.
-    mappable_list: list (default [])
-        empty list which fills with field values from color axis used for initial subplot, used to scale multiple plots along the same color axis. Used to make all 3x2 plots under a uniform color scale
+    mappable_list : list, default []
+        empty list which fills with field values from color axis used for
+        initial subplot, used to scale multiple plots along the same color
+        axis. Used to make all 3x2 plots under a uniform color scale
```
Comment (jmfranck) [inline pyspecdata/plot_funcs/pcolormesh.py:43]: again, fixing it to be to spec (see also [here](https://github.com/numpy/numpydoc/issues/284)).

note also that I'm doing `gq` to wrap the text to a reasonable width.

**Pull request merged**